### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by smontana30**
I replace all of the floats and clear trick with flexbox stuff and I had a few issues with flex but I think I was able to fix it and other issues.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,174 @@
...
+                        a lot fun and am always sad to return home.
+                        Vamos Colombia! 
+                    </p>
+                </div>
+            </div>
+
+            <!-- image on videos games -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img src="https://image-cdn.essentiallysports.com/wp-content/uploads/20200427185830/the-last-of-us-part-2-release-date.jpg" alt="The Last of Us Part II" >
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html]: Put the alt on a separate line so this tag isn't so long.
```
<img src="..."
     alt="...">

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,174 @@
...
+                        also very different. We are polar opposites of each other.
+                        My sister very arty. I'm very math and sciencey. 
+                    </p>
+                </div>
+            </div>
+
+            <!-- image on Miami where i grew up -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img src="https://blog-www.pods.com/wp-content/uploads/2019/08/MG_6_1_Miami.jpg" alt="City of Miami" >
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html]: Put alt on a separate line

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,174 @@
...
+                        his music and videos. Also, I went to his last concert 
+                        in Miami. It was insane, he was soooo good live 10/10
+                        recommend. 
+                    </p>
+                </div>
+            </div>
+
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://static.iris.net.co/semana/upload/images/2018/6/1/569673_1.jpg" alt="Colombian Soccer Team" >
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html]: Put alt on a separate line and remove extra space before "src"

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,18 +1,174 @@
...
+                        on the tv a lot. We alway watch parties during the Fifa World Cup
+                        to watch Colombia games. It was a lot of fun. Let's go 
+                        Colombia!
+                    </p>
+                </div>
+            </div>
+
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="images/shpe.png" alt="The Society of Professional Hispanic Engineers" >
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html]: Remove extra space before "src"

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,9 +1,180 @@
...
+    background-color: black;
+    color: white;
+    display: flex;
+    flex-direction: row;
+    font-size: 24px;
+    overflow: hidden;
+    position: sticky;
+    text-align: center;
+    width: 100%;
+
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/style.css]: Remove extra lines of whitespace

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -1,18 +1,174 @@
...
+            <a href="#box">Color Picker</a>
+        </nav>
+
+    <div class="header"> 
+        <div class="header-text"> 
+            <h1>Sofia Montana</h1>
+        </div>
     </div>
+
+    <div id="about-content">
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html:27]: Use a consistent style when choosing between "id" or "class". "header" above is a class while this is an id. I'd go with just making everything a class unless it needs to be an id to be used with the JavaScript (like "box" and "color").

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -47,7 +47,8 @@ <h2> Fun Fact Gallery </h2>
             <!-- image on Colombia -->
             <div class="column"> 
                 <div class="content"> 
-                    <img src="https://media.nomadicmatt.com/colombiaguide.jpg" alt="Colombia" >
+                    <img src="https://media.nomadicmatt.com/colombiaguide.jpg" 
+                    alt="Colombia" >
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html]: Two last things here, alt should be indented from img, and you don't need an extra space at the end before the closing bracket.
For example:

```
<img src="..."
     alt="...">
```

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -101,7 +103,8 @@ <h3>Being a Twin</h3>
             <!-- image on Miami where i grew up -->
             <div class="column"> 
                 <div class="content"> 
-                    <img src="https://blog-www.pods.com/wp-content/uploads/2019/08/MG_6_1_Miami.jpg" alt="City of Miami" >
+                    <img src="https://blog-www.pods.com/wp-content/uploads/2019/08/MG_6_1_Miami.jpg" 
+                    alt="City of Miami" >
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html]: Same here

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -115,7 +118,8 @@ <h3>Miami</h3>
             <!-- image on bad bunny my fav artist  -->
             <div class="column"> 
                 <div class="content"> 
-                    <img src="https://cdn3.pitchfork.com/longform/976/badbunnyheader.jpg" alt="bad bunny" >
+                    <img src="https://cdn3.pitchfork.com/longform/976/badbunnyheader.jpg" 
+                    alt="bad bunny" >
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html]: Here too

File: portfolio/src/main/webapp/index.html [hunk 10]
```diff
@@ -129,7 +133,8 @@ <h3>Bad Bunny</h3>
 
             <div class="column"> 
                 <div class="content"> 
-                    <img  src="https://static.iris.net.co/semana/upload/images/2018/6/1/569673_1.jpg" alt="Colombian Soccer Team" >
+                    <img src="https://static.iris.net.co/semana/upload/images/2018/6/1/569673_1.jpg" 
+                    alt="Colombian Soccer Team" >
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html]: Here too

**Review by mbehrlich — APPROVED** [review]

**Review by NicholasCobbFromGoogle — APPROVED** [review]

**Pull request merged**